<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="5,3" columnstretch="0,0" rowminimumheight="0,0">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupAll">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>任务目标池</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupTask">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>记忆缓存分解池</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupClient">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>终端管理</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupLog">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>处理日志记录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>服务管理</string>
    </property>
    <addaction name="Act_startsystem"/>
    <addaction name="Act_startsoap"/>
    <addaction name="Act_startloop"/>
    <addaction name="separator"/>
    <addaction name="Act_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>数据管理</string>
    </property>
    <addaction name="Act_listbrush"/>
    <addaction name="separator"/>
    <addaction name="Act_addtask"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>模块管理</string>
    </property>
    <addaction name="Act_model"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>程序设置</string>
    </property>
    <addaction name="Act_syscfg"/>
    <addaction name="Act_help"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="Act_startsystem">
   <property name="text">
    <string>启动系统</string>
   </property>
  </action>
  <action name="Act_startsoap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>启动服务器</string>
   </property>
  </action>
  <action name="Act_startloop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>启动调度</string>
   </property>
  </action>
  <action name="Act_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="Act_listbrush">
   <property name="text">
    <string>列表刷新</string>
   </property>
  </action>
  <action name="Act_addtask">
   <property name="text">
    <string>添加任务</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>相关测试</string>
   </property>
  </action>
  <action name="Act_syscfg">
   <property name="text">
    <string>系统设置</string>
   </property>
  </action>
  <action name="Act_help">
   <property name="text">
    <string>帮助说明</string>
   </property>
  </action>
  <action name="Act_model">
   <property name="text">
    <string>操作模块</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Act_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
